server/conf/application.conf

```diff
@@ -83,6 +83,10 @@ staging_ti_notification_mailing_list = ${?STAGING_TI_LIST}
 staging_applicant_notification_mailing_list = "seattle-civiform-applicants-notify@google.com"
 staging_applicant_notification_mailing_list = ${?STAGING_APPLICANT_LIST}
 
+## Common Intake More Resources HREF
+common_intake_more_resources_href = "https://access.arkansas.gov/Learn/Home"
+common_intake_more_resources_href = ${?COMMON_INTAKE_MORE_RESOURCES_HREF}
+
```

Comment:
**thought (blocking):** we need to ensure this gets thoroughly communicated to civic entities. Please include the env var in the release notes section.

Also need to add COMMON_INTAKE_MORE_RESOURCES to the variables passed through the deployment system.

Reply:
Added in https://github.com/civiform/cloud-deploy-infra/pull/137